flowermd/library/forcefields.py

```diff
@@ -481,3 +481,61 @@ def _check_widths(self):
                         "number of points for table energies and forces."
                     )
         return bond_width, angle_width, dih_width
+
+
+class EllipsoidForcefield(BaseHOOMDForcefield):
+    """A forcefield for modeling anisotropic bead polymers."""
+
+    def __init__(
+        self,
+        epsilon,
+        lper,
+        lpar,
+        bead_length,
```

Comment:
This is carried over from polyellipsoid, we should probably remove this and use lpar anywhere we need the bead length.

Reply:
@marjanAlbouye for consistency, what do you think about using `lpar` in both the `EllipsoidChain` class and `EllipsoidForcefield` class? Or, should we keep `bead_length` in `EllipsoidChain` and only use `lpar` in the forcefield, since it is one of the parameters needed by Gay-Berne?

Reply:
Either is fine. Probably it's better to use `lpar` (or even the complete name "len_parallel") in all cases.

Reply:
After some discussion we decided to use `lpar` in the `EllipsoidChain` class as well where it is half of the bead length along its major axis.